.github/workflows/fsp-qemu.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         (
           cd qemu/build-riscv64
           ../configure --target-list=riscv64-softmmu
-          make -j16
+          make -j16 mainboard
```

Comment:
this is the skipped make for QEMU itself, which wouldn't have a `mainboard` target

Reply:
fixed.